### PR TITLE
Create music album class

### PR DESCRIPTION
**Pull request by adel-gu**
# In this PR I made the following changes

- Create `MusicAlbum` class.
- Implement the `can_be_archived?()` in the MusicAlbum class, it returns true if both parent `can_be_archived?()` method and `on_spotify` variable are true, return false otherwise
- Add unit tests for all implemented methods.

### CODE REVIEW

File: classes/music_album.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, on_spotify: true)
+    super(publish_date)
```
Comment (RichardChileya) [inline classes/music_album.rb:7]: nice implementation of the music_album class

Comment (adel-gu) [reply classes/music_album.rb:7]: Thank you @RichardChileya 👏

**Review by RichardChileya — APPROVED** [review]
### APPROVED!!! 🎉 🎉 🎉 🍾 🍾 🍾
![]https://media.giphy.com/media/349qKnoIBHK1i/giphy.gif)
Hi @adel-gu  , 
Good job, this pull request has been approved!!!
Top Highlights! 👏 🟢
✅ Descriptive pull request message
✅ Commit messages are meaningful.

**Comment by adel-gu** [comment]
Thanks for reviewing my code @RichardChileya 👏👏👏

**Pull request merged**